proto/server/configEvtHandler.go

```diff
@@ -196,6 +182,22 @@ func handleDeviceGroupPost(configMsg *configmodels.ConfigMessage, subsUpdateChan
 	rwLock.Unlock()
 }
 
+func handleDeviceGroupDelete(configMsg *configmodels.ConfigMessage, subsUpdateChan chan *Update5GSubscriberMsg) {
+	rwLock.Lock()
+	if factory.WebUIConfig.Configuration.Mode5G {
+		var config5gMsg Update5GSubscriberMsg
+		config5gMsg.Msg = configMsg
+		config5gMsg.PrevDevGroup = getDeviceGroupByName(configMsg.DevGroupName)
+		subsUpdateChan <- &config5gMsg
+	}
```

Comment:
In the previous code, the deletion operation (device group or slice) takes place first and then, the writing event on channel happens. Now, here, in the new functions, you first write event to channel and then do the delete operation(s). Should we keep the same order as before (delete operation(s) and then write event to channel)? or is the order not relevant?

Reply:
One more thing, in the previous code the writing event to channel happened only once. That is, delete device group (if needed), then delete slice (if needed) and then write event to channel. With the current changes, if there is a delete device group and delete slice, we would be writing  the event to channel twice. Is this the intention with these changes?

Reply:
I don't think we're gonna write to the channel twice, because I removed the code which was originally responsible for doing that (lines 151-155). Now the handler functions are exclusively responsible for writing to the channel.

As for the order, it's a good question. I was thinking about the same. I decided to do it like that to be consistent with the rest of the handler functions. It's probably not critical, but I think it would make more sense to write to the channel after the operation is done; for all the handlers. How bout we take this to a separate PR?

Reply:
> I don't think we're gonna write to the channel twice, because I removed the code which was originally responsible for doing that (lines 151-155). Now the handler functions are exclusively responsible for writing to the channel.

Yes, you removed the original write to channel code and put it inside each function (delete device group and delete slice). My point is that if delete device group is true and delete slice is true, you would be writing to channel twice. However, this was not the behavior before. That is, before if delete device group was true and delete slice was true, writing to channel happened only once. Do you agree?

Reply:
> Please correct if I'm missing something, but I don't think we can be deleting a device group and a network slice at the same time. Especially after I added additional conditions to the `ifs`. If I'm not mistaken this would be 2 separate events.

I do not know whether it is doable to have both `delete device-group` and `delete slice` as a single operation. My comment was primary comparing of what the code is currently doing and the proposed changes in your PR.

@patriciareinoso, just in case, do you any comment/feedback about the changes proposed in this PR (especially, about whether a `delete group-device` and `delete slice` could be called in the same operation)? Thanks!

Reply:
Following the execution from `webconsole/configapi/api_default.go`, my understanding is that Delete network slice and Delete device group operations are always handled as separate events. We will never have a `delete slice` and `delete device-group` at the same time in the  `configChannel` message. 
This being said, there is no change in behavior, we used to send 1 `Update5GSubscriberMsg` messages per event and we still do so after modifications.
I agree that using `if` instead of `else if` in lines 130 and 135 can be misleading.

Reply:
@patriciareinoso, thanks for your feedback. The code before, first executed delete operation (device group or slice) and then writes to channel. With this PR, it happens the opposite, first write to channel and then execute delete operation. Any thoughts on this? Does this change have any side-effects? or does it not make any difference?

Reply:
The order does not matter. The processing of the message in the channel is completely independent from the DB operations. I would keep the same order in the delete and post operations to make it consistent. 
I also think that extracting `handleNetworkSliceDelete` and `handleDeviceGroupDelete` methods is a good idea, it's more readable.